pom.xml

```diff
@@ -192,6 +193,12 @@
         <artifactId>junit-jupiter</artifactId>
         <version>${testcontainers.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-compress</artifactId>
+        <version>${commpress.version}</version>
+        <optional>true</optional>
```

Comment:
@arturobernalg I do not think this has any effect. Anyway, `dependencyManagement` section is the wrong place to declare artifact's optionality. This needs to be declared in the module's `dependecies` section, not here.